packages/api-server/src/index.ts

```diff
@@ -1,6 +1,41 @@
 #!/usr/bin/env node
 import yargs from 'yargs'
 
-import { apiCliOptions, apiServerHandler } from './cliHandlers'
+import {
+  apiCliOptions,
+  webCliOptions,
+  commonOptions,
+  apiServerHandler,
+  webServerHandler,
+  bothServerHandler,
+} from './cliHandlers'
 
-apiServerHandler(yargs.options(apiCliOptions).argv)
+const commandPath = yargs.argv.$0
+
+const positionalArgs = yargs.argv._
+
+// "bin": {
```

Comment:
This'll go out of date very quickly! :D

Reply:
I mean, it it goes out of date hopefully that means we're re-architecting index.js to be a "real" command. But that said, is `yargs.argv.slice(2)` a better way to handle this?

Reply:
Played with this a bit and couldn't find anything else satisfying enough to replace it. yargs.argv returns two default objects: 
1) `_`: array of args without a corresponding option (i.e. positionals)
2. `$0`: script path (aka command name)

If someone tries something like `yarn rw-server --side api`, this won't work because the Yargs object will include `side: 'api'`. But it shouldn't work in that case anyway.

Given the current structure, I'm ok with this for now. Resolving conversation but please feel free to re-open.

Reply:
Yep fine by me DP - my comment was actually about the comment with the bins listed, no big deal. I think I'd prefer not to have the bins listed in a comment here.